storage/persistence_test.go

```diff
@@ -51,3 +67,138 @@ func TestUploadTestMetadata(t *testing.T) {
 	expectedMetadata := `{"id":"62dba0159041307f697e6ccc","name":"test0","build_id":"5a75f537726934e4b62833ab6d5dca41","task_id":"t0"}`
 	assert.JSONEq(t, expectedMetadata, string(contents))
 }
+
+// Checks that we stored the data in the right location and text format
```

Comment:
Nit: remove this, no need to add function documentation in tests.

Reply:
I don't 100% agree with never adding function documentation in tests, but in this case it's a very "meh" and not super informative comment so I removed it.